app/views/tube_rack_statuses/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="page-header">
+    <h1>Tube Rack Status</h1>
+</div>
+
+<%= pagination @tube_rack_statuses %>
+
+<table class="table">
+  <thead>
+    <th>Tube rack</th>
+    <th>Status</th>
+    <th>Plate</th>
```

Comment:
Feel like a more accurate heading would be 'Descendants'? Because a) there might be more than 1, and b) they might not be plates (in future)

Also wondering whether this is the right place to show descendants - was thinking about it as just showing whether or not the tube rack was imported successfully.